tsconfig.json

```diff
@@ -20,7 +20,7 @@
     "paths": {
       //      "types": [ "./types/*", "./node_modules/@types"],
       "*": [ "./types/*", "./node_modules/@types"]
-    }
+    },
```

Comment:
Trailing commas are not supported in common json syntax

Reply:
Fixed